modules/index.js

```diff
@@ -23,6 +23,9 @@ export * from './validations/index';
 // This is only done in testing because of the performance penalty.
 export let debug = false;
 
+/** @param {boolean} newValue */
+export const setDebug = (newValue) => { debug = newValue; };
```

Comment:
this is required because mutable export bindings (`export let ...`) have a lot of caveats in ESM, and don't work well with our setup